homeassistant/components/proxmox/binary_sensor.py

```diff
@@ -0,0 +1,54 @@
+import homeassistant.components.proxmox as proxmox
+from homeassistant.components.switch import SwitchDevice
```

Comment:
This is the binary sensor not the switch.

Reply:
Fixed.